app/models/eligibilities/eight_a_stars.rb

```diff
@@ -0,0 +1,8 @@
+module Eligibility
```

Comment:
I've been namespacing on one line in this project, so `Eligiblity::EightaStars` . Not hugely different but helpful for knowing that a module is for namespacing rather than including behavior, imo